<launch>

    <!-- Find and include the generic XBox controller launchfile from our thunderbots package -->
    <include file="$(find thunderbots)/launch/xbox_controller.launch" />

    <!-- Launch the grsim_communication node -->
    <node name="grsim_communication" pkg="thunderbots" type="grsim_communication" output="screen">
    </node>

</launch>
